### PR TITLE
Ayelet binder/pdf download

### PR DESCRIPTION
**Pull request by ayeletBinder**
i want you allow my code.
this code is pdf download for the reports files.
thank you.

### CODE REVIEW

**Review by Chani-Pielet — APPROVED** [review]

File: client-side/src/components/pdf.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      { title: "Site name", dataKey: "SiteName" },
+      { title: "Browsing time", dataKey: "BrowsingTime" },
+      { title: "Avg. for a day", dataKey: "AvgForADay" },
+    ];
+    doc.autoTable({
+      columns: columns,
+      body: data,
+      startY: 35,
+    });
+    doc.text("developed by kamatech", 10, 250);
```
Comment (SariGlick) [inline client-side/src/components/pdf.jsx:21]: ExtraTech :)

File: client-side/src/components/pdf.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    doc.save("timeOut-Report.pdf");
+  };
+
+  return (
+    <div>
+      <button onClick={generatePdf}>Download as PDF</button>{/* לקחת כפתור מה UI  */}
```
Comment (SariGlick) [inline client-side/src/components/pdf.jsx:27]: the button already exists, you can replace it

File: client-side/src/components/pdf.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import jsPDF from 'jspdf';
+import 'jspdf-autotable';
```
Comment (SariGlick) [inline client-side/src/components/pdf.jsx:3]: wrong import

File: client-side/src/index.js [hunk 4]
```diff
@@ -3,11 +3,17 @@ import ReactDOM from 'react-dom/client';
 import './index.scss';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
```
Comment (SariGlick) [inline client-side/src/index.js:6]: change imports order, first external modules, and then internal files

File: client-side/src/stories/header/header.scss [hunk 5]
```diff
@@ -0,0 +1,34 @@
+@import '../stylies.scss';
+.navbar{
+    background-color: $primary-color  ;
+    color: $secondary-color  ;
+}
+.logo{
+    margin-right: 2 ;
+    font-family: 'Nanum Gothic' ;
+    color: rgb(103, 252, 210) ;
```
Comment (SariGlick) [inline client-side/src/stories/header/header.scss:9]: move color to variable

File: client-side/src/stories/header/header.scss [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+}
+.menu{
+     display: 'block' ;
+     math-depth: 'none'  ;
+}
+.left-side-box{
+    flex-grow: 1;
+    display: flex;
+  
+    @media (min-width: 768px) {
```
Comment (SariGlick) [inline client-side/src/stories/header/header.scss:20]: change all px and vw to %

File: client-side/src/stories/tabs/tabs.stories.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import LabTabs from './tabs';
+
+export default {
+    title:"tab",
+    component:"LabTabs",
+    argsTypes:{onclick:{action:"lll"}}
+}
```
Comment (SariGlick) [inline client-side/src/stories/tabs/tabs.stories.js:8]: what is the 'lll'?

File: client-side/src/stories/tabs/tabs.stories.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+export const widthtabs=Template.bind({})
+widthtabs.args={
+    classNameOfTags:"width-tabs",
+}
+
+export const footerTabs=Template.bind({})
+footerTabs.args={
+    nameOfClass:"footer-tabs",
+    text:["Kamatech"],
```
Comment (SariGlick) [inline client-side/src/stories/tabs/tabs.stories.js:27]: ExtraTech

**Review by SariGlick — CHANGES_REQUESTED** [review]

File: client-side/src/stories/tabs/tabs.stories.js (hunk 8 above)
Comment (ayeletBinder) [reply client-side/src/stories/tabs/tabs.stories.js:27]: OK

File: client-side/src/components/pdf.jsx (hunk 1 above)
Comment (ayeletBinder) [reply client-side/src/components/pdf.jsx:21]: OK

File: client-side/src/components/pdf.jsx (hunk 2 above)
Comment (ayeletBinder) [reply client-side/src/components/pdf.jsx:27]: OK

File: client-side/src/components/pdf.jsx (hunk 3 above)
Comment (ayeletBinder) [reply client-side/src/components/pdf.jsx:3]: איפה זה לא תקין?
זה? import 'jspdf-autotable';

כי הוא נותן לי לייבא ככה

File: client-side/src/index.js (hunk 4 above)
Comment (ayeletBinder) [reply client-side/src/index.js:6]: ok

File: client-side/src/stories/header/header.scss (hunk 5 above)
Comment (ayeletBinder) [reply client-side/src/stories/header/header.scss:9]: ok

**Pull request closed without merging**